test/Microsoft.DotNet.SourceBuild.Tests/assets/SdkContentTests/ApiDiff.suppression

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
```

Comment:
The end user UX of this diff files is fairly important IMO.  In other words, when there is a test failure, being able to interpret what is happening from the diff context alone is important.  With that in mind, the diagnostic Ids alone are not helpful.  Does APICompat have the ability to include the human readable description so that you don't have to lookup the id?

Reply:
Will check on better messages.